src/test/kotlin/com/hypto/iam/server/apis/OrganizationApiKtTest.kt

```diff
@@ -45,6 +47,37 @@ internal class OrganizationApiKtTest : AbstractContainerBaseTest() {
     private val gson: Gson by inject()
     private val passcodeService: PasscodeService by inject()
 
+    @Test
+    fun `create organization with valid root credentials and check login success`() {
+        withTestApplication(Application::handleRequest) {
+            val (createdOrganizationResponse, createdUser) = DataSetupHelper
+                .createOrganization(this)
+            val createdOrganization = createdOrganizationResponse.organization
+            val authString = "${createdUser.email.uppercase()}:${createdUser.password}"
+            // testing for case Insensitive Email
```

Comment:
This comment must be the test case name. Explain what case is being tested using case function name